client/js/components/statement/splitStatement/SideBar.vue

```diff
@@ -52,7 +52,7 @@
 
       <div
         v-else
-        class="flex flex-col">
+        class="flex flex-col w-12">
```

Comment:
does `w-full` do the job as well?

Reply:
Yes
Done in https://github.com/demos-europe/demosplan-core/pull/3562/commits/4ab7c18a4fb420be4b6ef2a2a69d10be8f477c55